### PR TITLE
Added vulnerabilities badge on README.md and docs/PT-BR/README.md

### PR DESCRIPTION
**Pull request by jzandag**
## Pull Request Checklist

- [x] Have you followed the steps in our [Contributing](https://github.com/wlsf82/protractor-helper/blob/master/docs/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/wlsf82/protractor-helper/pulls) for the same update/change?
- [ ] Have you successfully ran tests with your changes locally?
- [ ] Have you written new tests for your changes, if necessary?
- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you made improvements to the documentation, if necessary? ([docs/](https://github.com/wlsf82/protractor-helper/tree/master/docs) folder and [README.md](https://github.com/wlsf82/protractor-helper/blob/master/README.md))
- [x] Have you linked the correct [GitHub issue](https://github.com/wlsf82/protractor-helper/issues), if there is any?

Linked issue: #132, added vulnerabilities badge on README.md and docs/PT-BR/README.md. also updated the credits section

> Pull requests that do not pass the coding static analysis and the automated tests on [Github Actions](https://github.com/wlsf82/protractor-helper/actions), and the code quality verification on [Better Code Hub](https://bettercodehub.com/) will not be reviewed.

### CODE REVIEW

File: docs/PT-BR/README.md [hunk 1]
```diff
@@ -343,7 +343,7 @@ O **protractor-helper** foi criado por [Walmyr Filho](https://walmyr.dev) e é m
 
 Siga eles no Twitter ([@walmyrlimaesilv](https://twitter.com/walmyrlimaesilv) e [@paulorgoncalves](https://twitter.com/paulorgoncalves)).
 
-Agradecimentos ao [Lucas Amaral](https://www.linkedin.com/in/lopesdoamaral/), [Michiel Cuijpers](https://github.com/MichielCuijpers), [Ghazi Khan](https://codewithghazi.com/), [Pedro Hyvo](https://www.linkedin.com/in/pedrohyvo/), e [Urvi Koladiya](https://github.com/cp-urvi-k) por contribuírem com o projeto.
+Agradecimentos ao [Lucas Amaral](https://www.linkedin.com/in/lopesdoamaral/), [Michiel Cuijpers](https://github.com/MichielCuijpers), [Ghazi Khan](https://codewithghazi.com/), [Pedro Hyvo](https://www.linkedin.com/in/pedrohyvo/), [Urvi Koladiya](https://github.com/cp-urvi-k), e [Zidrex Andag](https://www.linkedin.com/in/john-zidrex-andag-3285b31a7/) por contribuírem com o projeto.
```
Comment (PauloGoncalvesBH) [inline docs/PT-BR/README.md:346]:
```suggestion
Agradecimentos ao [Lucas Amaral](https://www.linkedin.com/in/lopesdoamaral/), [Michiel Cuijpers](https://github.com/MichielCuijpers), [Ghazi Khan](https://codewithghazi.com/), [Pedro Hyvo](https://www.linkedin.com/in/pedrohyvo/), [Urvi Koladiya](https://github.com/cp-urvi-k) e [Zidrex Andag](https://www.linkedin.com/in/john-zidrex-andag-3285b31a7/) por contribuírem com o projeto.
```

**Review by PauloGoncalvesBH — CHANGES_REQUESTED** [review]

File: docs/PT-BR/README.md (hunk 1 above)
Comment (jzandag) [reply docs/PT-BR/README.md]: done @PauloGoncalvesBH

**Comment by wlsf82** [comment]
Hi, @jzandag, do you mind rebasing your branch on top of the most up-to-date version of the upstream repo and force a push, please?

We had to make a temporary change in an npm script to allow the tests to work properly.

Sorry for the inconvenience.

**Comment by wlsf82** [comment]
> Hi, @jzandag, do you mind rebasing your branch on top of the most up-to-date version of the upstream repo and force a push, please?
> 
> We had to make a temporary change in an npm script to allow the tests to work properly.
> 
> Sorry for the inconvenience.

You know what, nevermind. I'll rebase and merge, and it should solve the issue.

**Review by wlsf82 — APPROVED** [review]
Hi, @jzandag,

You've got a 🍰!

> The test failure is not related to your change, and the fix for that is already on the main branch, so rebasing and merging will fix it.

Thanks for your contribution!

**Comment by jzandag** [comment]
Thank you! Actually I was trying to rebase my fork, may I know what the process would be for that?

**Comment by wlsf82** [comment]
> Thank you! Actually I was trying to rebase my fork, may I know what the process would be for that?

@jzandag, like this https://github.com/wlsf82/protractor-helper/pull/129#pullrequestreview-572289526

**Comment by jzandag** [comment]
Thank you @wlsf82! learning something new for the day!

**Pull request merged**